schema.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE cheesecake;
+USE cheesecake;
+
+CREATE TABLE user(
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    role VARCHAR(255) NOT NULL,
+    PRIMARY KEY (id)
+)
+
+CREATE TABLE job(
```

Comment:
rename table to `jobs`

The keys in the [data.json](https://github.com/anonair1920/CheeseCake/blob/master/data.json#L3-L7) file should match here.

missing

- [ ] company
- [ ] description 
- [ ] mail